code/style.css

```diff
@@ -1,58 +1,264 @@
+  letter-spacing: 4px;
+  text-align: center;
+}
+
+h3 {
   font-weight: 300;
-  background-color: #fff;
+  font-family: 'Abril Fatface';
+  color: white;
+  font-size: 18px;
+  width: 80%;
+  margin: auto;
+  margin-top: 10px;
+  font-style: italic;
+  letter-spacing: 1px;
+}
+
+h4 {
+  font-weight: 0;
+  font-family: 'Abril Fatface';
+  color: rgb(225, 128, 10);
+  font-size: 28px;
+  text-align: center;
+  margin-top: auto;
+  letter-spacing: 1px;
+  font-style: italic;
+  margin-bottom: auto;
+}
+
+h5 {
+  font-weight: 0;
+  font-family: ARIAL;
+  color: black;
+  font-size: 24px;
+  text-align: center;
+  margin-top: 20px;
+  margin-bottom: 5px;
+  width: 80%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+h5.reveal {
+  color: white;
+  font-size: 18px;
+}
+
+p {
+  font-weight: 0;
+  font-family: 'Abril Fatface';
+  color: white;
+  font-size: 24px;
+  margin-top: -5px;
+  letter-spacing: 1px;
+  margin-bottom: 17px;
 }
 
 .cards {
-  margin: 50px auto;
+  margin: 10px auto;
   width: 90%;
-  height: 600px;
-  background: #ccc;
+  height: 440px;
+  background: rgb(121, 3, 3);
   display: flex;
   justify-content: space-around;
   align-items: center;
+  margin-bottom: 20px;
+  margin-top: 20px;
 }
 
 .killer-deck {
   width: 250px;
   height: 400px;
-  background: #000;
+  background: rgb(18, 70, 28);
   text-align: center;
-  padding-top: 30px;
+  padding-top: 27px;
   border-radius: 16px;
+  box-shadow: 3px 3px 10px rgb(41, 26, 26);
+}
+
+.killerImage {
+  border-radius: 26px;
 }
 
 .weapon-deck {
   width: 250px;
   height: 400px;
-  background: rgb(10, 53, 10);
+  background: rgb(18, 70, 28);
   text-align: center;
-  padding-top: 30px;
+  padding-top: 27px;
   border-radius: 16px;
+  box-shadow: 3px 3px 10px rgb(41, 26, 26);
 }
 
 .room-deck {
   width: 250px;
   height: 400px;
-  background: midnightblue;
-
+  background: rgb(38, 90, 113);
   text-align: center;
-  padding-top: 30px;
+  padding-top: 27px;
   border-radius: 16px;
+  box-shadow: 3px 3px 10px rgb(41, 26, 26);
 }
 
 button {
   padding: 8px 16px;
-  background: crimson;
+  background: rgb(216, 119, 0);
+  color: #fff;
   border: none;
   font-size: 20px;
   border-radius: 8px;
   display: block;
-  margin: 0 auto;
+  margin: auto;
+  font-family: 'Abril Fatface', cursive;
+  letter-spacing: 1px;
+  margin-bottom: 25px;
+  margin-top: 17px;
+}
+
+@media (max-width: 900px) and (min-width: 651px) {
+
+  h1 {
+    font-size: 65px;
+  }
+
+  h2 {
+    font-size: 14px;
+    margin-bottom: 6px;
+    margin-top: -12px;
+  }
+
+  p {
+    font-size: 20px;
+    margin-bottom: 18px;
+  }
+
+  .cards {
+    margin-bottom: -22px;
+    margin-top: auto;
+  }
+
+  .killerImage {
+    margin-top: -10px;
+    margin-bottom: -8px;
+  }
+
+  .killer-deck {
+    width: 250px;
+    height: 350px;
+    margin: 10px;
+  }
+
+  .weapon-deck {
+    width: 250px;
+    height: 350px;
+    margin: 10px;
+  }
+
+  .room-deck {
+    width: 250px;
+    height: 350px;
+    margin: 10px;
+  }
```

Comment:
Great job with the responsiveness.  Oftentimes it's very difficult to reduce the amount of CSS when designing for responsiveness.